src/node_task_runner.cc

```diff
@@ -221,7 +221,12 @@ void ProcessRunner::Run() {
         return;
       }
     }
-
+    // Adding node_modules/.bin under cwd to PATH environment variable
+    std::filesystem::path nmBin =
+        std::filesystem::path(cwd) / "node_modules" / ".bin";
+    if (std::filesystem::is_directory(nmBin)) {
+      path_env_var_ = nmBin.string() + env_var_separator + path_env_var_;
+    }
```

Comment:
@anonrig I tried to process the path given with --run-from and add the ```node_modules/.bin``` directory under the cwd to the ```PATH``` Without changing the cwd of the parent process, only the cwd of the spawned process will be set, is this a good solution ?